src/app/views/authentication/auth-util-components/UtilComponents.tsx

```diff
@@ -1,21 +1,33 @@
-import { PrimaryButton, Stack } from 'office-ui-fabric-react';
+import { IconButton, PrimaryButton, Stack } from 'office-ui-fabric-react';
 import React from 'react';
 import { FormattedMessage } from 'react-intl';
 import Profile from '../profile/Profile';
 
-export function showSignInButtonOrProfile(tokenPresent: boolean, mobileScreen: boolean, signIn: Function) {
+export function showSignInButtonOrProfile(
+  tokenPresent: boolean,
+  mobileScreen: boolean,
+  signIn: Function,
+  minimised: boolean
+) {
+
+  const signInButton = minimised ? <IconButton
```

Comment:
The `<PrimaryButton />` is **duplicated** on the second part of the ternary operator. We can use a **function** to **dry** it up by passing `minimised` as an argument, then use `minimised` to control showing the sign in message.